lib/lib-storage/src/Upload.ts

```diff
@@ -29,9 +31,6 @@ export interface RawDataPart {
 interface UploadedPartAttributes {
   PartNumber: number;
   ETag: string;
-  ChecksumCRC32?: string;
-  ChecksumCRC32C?: string;
-  ChecksumSHA1?: string;
```

Comment:
Since we're not currently supporting or using these checksum options, I thought it'd be better just to remove them here and below, to minimize changes to this forked lib and potential confusion or errors.